leptos_hotkeys/src/context.rs

```diff
@@ -177,3 +187,11 @@ where
 pub fn use_hotkeys_context() -> HotkeysContext {
     use_context::<HotkeysContext>().expect("expected hotkeys context")
 }
+
+#[cfg(not(feature = "ssr"))]
+fn clean_key(event: &web_sys::KeyboardEvent) -> String {
+    match event.key().as_str() {
```

Comment:
@friendlymatthew When I merged in the changes from #121, I noticed that this is the only place where `event.key()` is referenced. So if you've come up with a feature flag for switching between `event.code()` and `event.key()`, I'll add it here.

Reply:
@maxbergmark thanks for flagging! Would you be interested in attacking #131? Work is taking up all my time, so I'm afraid I won't have time to get this done ASAP.

On another note, would you be interested in joining as a maintainer? You'd be requested for reviews and help steer this project. Let me know! :)

Reply:
@friendlymatthew I made an attempt at solving #131. I added a feature called `event_key` and just used it in the `clean_key` function. That should do the job, but I'd be happy to fix any issues that I might have overlooked (maybe the spacebar handling?). The name of the feature flag was just something I made up, I'll change it if you have a better suggestion.

I'm honored to be invited as a maintainer of this repo, but right now I can't find the time that it would require. I think you're doing great work, and I'll happily keep using the repo, and contribute in discussions and issues when I have some time left over.